contents/founders/how-come-we-ship-so-much.md

```diff
@@ -0,0 +1,138 @@
+
+We invested considerable time getting our design *system* up and running, and worked with a designer to get it done. Then we moved onto this "no design by default" phase. Had we been in this phase from the beginning, the app would have been very inconsistent... at best.
+
+Today, we have people that can help engineers with design, but they are very happy to work reactively and fast – and they don't block merging work during QA. This means we do ship work where the design isn't perfect, but we won't drop things that are important and will keep improving them.
+
+For example, we've redesigned this entire website ~5 times.
+
+### 2. No product management by default
+
+The responsibilities of a product manager still exist, but our engineers:
+
+* Talk to users.
+
+* Decide what to build within their product (and sometimes pitch whole new products).
+
+* Have complete access to metrics like revenue, hosting costs, and (soon) staff costs, for their product.
+
+The concept internally is a "[product engineer](/blog/what-is-a-product-engineer)" – i.e. an engineer who is product-minded and autonomous, rather than mindlessly filing tickets.
+
+We have one product manager who steps in reactively if a small team needs more support.
+
+The above requires a lot of context setting from the company. To that end, we're *unusually transparent*. We share board slides, financial information, fundraising progress before, during and after it happens, and much more.
+
+### 3. Hiring with intention
+
+I think 90%+ of an organization's problems are solved by hiring right – see [what we've learned about hiring](https://newsletter.posthog.com/p/everything-weve-learned-about-hiring) in our Substack newsletter.
+
+We've found it invaluable to pay people during our hiring process to do some actual work. Over time, we've refined this down to a one-day task, which gives us a good sense of how much someone can get done and the quality of their actual code.
+
+We tend to hire people with more experience, who we find work better with more autonomy. I think this is the magic combo when you're an all remote company, though we're working on ways to get better at growing people who are earlier in their careers.
+
+### 4. Each team is a mini startup
+
+I went to a superb talk by [Jeff Lawson](https://www.linkedin.com/in/jeffiel), who runs [Twilio](https://twilio.com/). He described how the greatest innovation at [Amazon](https://www.amazon.com/) was that it felt like thousands of small startups. Simply speaking, a startup gets more done per person than a big corporate.
+
+PostHog works like a group of startups with lots of [small teams](/handbook/company/small-teams). Each of our products has a team of up to 6 who can ship with minimal interference from the rest of the company. We hire a lot of former technical founders to help here.
```

Comment:
This point doesn't tell me what we mean by a "mini startup". Can we add more detail? how do our teams operate? What makes it like a mini startup?

Reply:
Good point. I can add this. Key points:

- They decide what to build
- They own their own pricing
- Responsible for monitoring churn, user feedback etc.